.agents/skills/release/SKILL.md

```
@@ -0,0 +1,162 @@
+---
+name: release
+description: Run the MistKit release runbook — verify the vX.Y.Z release branch is green, assemble the ReleaseNotes.md section and README roadmap entry, merge to main, tag as X.Y.Z, publish the GitHub pre-release, and roll the Examples' MISTKIT_BRANCH pins. Use when asked to cut, prepare, or ship a release.
+argument-hint: "Which release? e.g. v1.0.0-beta.5"
+disable-model-invocation: true
+---
+
+# MistKit Release Runbook
+
+## The naming rule
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)